src/pipx/commands/ensure_path.py

```diff
@@ -97,7 +97,7 @@ def ensure_path(location: Path, *, force: bool) -> Tuple[bool, bool]:
 
 def ensure_pipx_paths(force: bool) -> ExitCode:
     """Returns pipx exit code."""
-    bin_paths = {constants.LOCAL_BIN_DIR}
```

Comment:
We probably need to add the global binary location here.

Reply:
Updated. The code became a bit ugly there, let me know if you have any suggestions.

Reply:
Reviewing this - maybe it's actually a good idea to only add the global location to the PATH if ensurepath is run with the `--global` flag, in which case no code change in the `ensurepath` command is necessary? That way you only need `sudo` to run `ensurepath`, if you want the global bins on your path.

I'd suggest adding that to the docs though. Sorry for sending you on that goose hunt.